src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -84,42 +79,18 @@ type CurrentRefinementsConnectorParams = {
   transformItems?: (items: Item[]) => any;
 };
 
-export type CurrentRefinementsRendererOptions<
-  TCurrentRefinementsWidgetParams
-> = {
+export type CurrentRefinementsRendererOptions = {
   items: Item[];
   refine(refinement: ItemRefinement): void;
```

Comment:
Noted for a future pull request.

Reply:
@Haroenv This vocabulary is confusing but `ItemRefinement` makes more sense in this context because an item contains refinements:

```
item
 - refinement
 - refinement
```